acceptance/indy_acceptance/indy-tests/libraries/constant.py

```diff
@@ -0,0 +1,60 @@
+
+"""
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+
+Containing all constants that are necessary to execute test scenario.
+"""
+
+
+class Colors:
```

Comment:
This class must be a subclass of `Enum`.

Reply:
Updated. Please refer to the latest code.